lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -758,9 +758,9 @@ TF_DataType TFGraphLowering::getTensorFlowDataType(SILType type,
 // Helpers to create TensorFlow graph nodes.
 //===----------------------------------------------------------------------===//
 
-static TF_Tensor *convertValuesToTensor(ArrayRef<SingleValueInstruction*> elts,
-                                        ArrayRef<int64_t> shape,
-                                        TF_DataType dtype) {
+static TF_Tensor *
+convertValuesToTensorLegacy(ArrayRef<SingleValueInstruction *> elts,
```

Comment:
Add a note here to explain why is legacy and when it should be removed?

Reply:
Done. Added:
```
// TODO: remove this function once we complete the migration to graph op inst.
```